Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN \
   --no-cache \
   --no-progress \
   --update \
+  --upgrade \
```

Comment:
Not too familiar with alpine - but wondering if there's a more general approach 🤔

I believe we want to update all depedencies, so maybe a separate step for `apk update && apk upgrade --no-cahce`?

Not entirely sure what `build-base` is and why we're adding that either, but maybe I'm just showing my ignorance here 😂

Reply:
Ideally, we want the image to be as small as possible. Keeping as few "layers" as possible (RUN lines) lets you squash the image size. I was trying to find a more official resource to explain this, but I could only find the following.
https://learnk8s.io/blog/smaller-docker-images

Reply:
RE build-base. It installs basics inside an image, like the GCC compiler, make etc.